aws/secrets.mk

```diff
@@ -1,5 +1,6 @@
 # Macroses
 ########################################################################################################################
+SERVICE_SECRETS_BACKUP_FILE ?= $(INFRA_DIR)/env/$(ENV)/secrets/$(SVC)_backup.json
```

Comment:
I would not use `_` as a separator, let's try to stick to dash or dot please.

Reply:
update with dash